src/rgw/rgw_op.cc

```diff
@@ -858,6 +858,37 @@ void RGWPutObjTags::execute()
     return;
   }
 
+  auto content_md5 = s->info.env->get("HTTP_CONTENT_MD5");
+
+  std::string content_md5_bin;
+  if (content_md5) {
+    ldout(s->cct, 15) << "content_md5=" << content_md5 << dendl;
+    try {
+      content_md5_bin = rgw::from_base64(boost::string_view(content_md5));
+    } catch (...) {
```

Comment:
Please don't do that. If you're going to signal a specific error catch that exception. I don't really mind if you catch std::exception instead, but catching ... without rethrowing can really mess up some platforms. (Including GNU if we ever use thread cancellation anywhere, though it has the worst effects on MS.)

Reply:
in this case, we definitely don't want to rethrow